views/interface.js

```diff
@@ -1,10 +1,26 @@
 document.addEventListener("DOMContentLoaded", function() {
 
-var storyComponent = new StoryComponent(story);
+	var api = "http://content.guardianapis.com/search?api-key=test"
 
- renderStory = function(storyComponent) {
- 		storyComponent.render();
- }
+	function getApi(api) {
```

Comment:
Can we do some better naming here? We have two api's to query so it might be good to have a distinction